### DISCUSSION TITLE
SSL Cert for apex domain and www

### DISCUSSION THREAD

**Discussion by whalley** [community/community#23496] [Other Feature Feedback, Questions, & Ideas]
<p>This seems to have been asked numerous times but it still does not seem to work for me (probably something stupid I have done).</p>
<p>Site is here: <a href="https://github.com/moneymanagerex/moneymanagerex.github.io" class="inline-onebox" rel="noopener nofollow ugc">GitHub - moneymanagerex/moneymanagerex.github.io: MMEX Home Site</a></p>
<p>CNAME file setup with <code>moneymanagerex.org</code> specified.</p>
<p>I have A records as below</p>
<pre><code class="lang-auto">moneymanagerex.org.	600	IN	A	185.199.110.153
moneymanagerex.org.	600	IN	A	185.199.109.153
moneymanagerex.org.	600	IN	A	185.199.111.153
moneymanagerex.org.	600	IN	A	185.199.108.153
</code></pre>
<p>and a CNAME of<br>
<code>www.moneymanagerex.org.	3600	IN	CNAME	moneymanagerex.github.io.</code></p>
<p><code>https://moneymanagerex.org</code> works fine but <code>https://www.moneymanagerex.org</code> flags a security risk as the cert generated just covers <code>moneymanagerex.org</code></p>
<p>Any clues?</p>

**Comment by jfullstackdev** [comment]
<p>this is on my end, seems things are okay</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/b/2b42521a2e0987190c00971959bee70d28c49659.png" data-download-href="/uploads/short-url/6aGCBEHr8xvKVrKyPZdOQohA7Cp.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181077410-7c077f88-a0db-4ff7-aded-013df2532e8a.png" alt="image" data-base62-sha1="6aGCBEHr8xvKVrKyPZdOQohA7Cp" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/b/2b42521a2e0987190c00971959bee70d28c49659_2_690x373.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/b/2b42521a2e0987190c00971959bee70d28c49659_2_1035x559.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/2/b/2b42521a2e0987190c00971959bee70d28c49659_2_1380x746.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/b/2b42521a2e0987190c00971959bee70d28c49659_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 201 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by whalley** [comment]
<p>Hmmm, This is what I get in Firefox when going to <a href="https://www.moneymanagerex.org" rel="noopener nofollow ugc">https://www.moneymanagerex.org</a>. Oddly <a href="http://www.moneymanagerex.org" rel="noopener nofollow ugc">http://www.moneymanagerex.org</a> works and re-directs to <a href="https://moneymanagerex.org" rel="noopener nofollow ugc">https://moneymanagerex.org</a>.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/e/1ee1a248c109b6e9859b14d9c2a2eafa8f228d09.png" data-download-href="/uploads/short-url/4pbK2pBGB7k5aycHVkxFWP5cqp3.png?dl=1" title="CleanShot 2022-05-19 at 08.51.53" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/487345/181077413-86e6e1b5-9c7d-42e2-8fa9-f1d2fb8b8610.png" alt="CleanShot 2022-05-19 at 08.51.53" data-base62-sha1="4pbK2pBGB7k5aycHVkxFWP5cqp3" width="690" height="444" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/e/1ee1a248c109b6e9859b14d9c2a2eafa8f228d09_2_690x444.png, https://aws1.discourse-cdn.com/github/original/3X/1/e/1ee1a248c109b6e9859b14d9c2a2eafa8f228d09.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/1/e/1ee1a248c109b6e9859b14d9c2a2eafa8f228d09.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/e/1ee1a248c109b6e9859b14d9c2a2eafa8f228d09_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">CleanShot 2022-05-19 at 08.51.53</span><span class="informations">947×610 53.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment]
<p>and this is using Edge, seems it’s blocking the JS, although the redirect is automatic</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/3/d35b4e83a9e9bd80b8f11da59b54d7f0e9438565.png" data-download-href="/uploads/short-url/u9KmW7N9e5MPdJvyJU8NICeuXfD.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181077417-7f471db6-c7d7-4228-ae1d-16d6c687a17f.png" alt="image" data-base62-sha1="u9KmW7N9e5MPdJvyJU8NICeuXfD" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d35b4e83a9e9bd80b8f11da59b54d7f0e9438565_2_690x373.png, https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d35b4e83a9e9bd80b8f11da59b54d7f0e9438565_2_1035x559.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d35b4e83a9e9bd80b8f11da59b54d7f0e9438565_2_1380x746.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d35b4e83a9e9bd80b8f11da59b54d7f0e9438565_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 123 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by yoannchaudet** [comment]
<p><a class="mention" href="/whalley">@whalley</a> This typically happen when your <code>www</code> DNS record wasn’t available yet to our servers (Pages) when we requested a TLS certificate on your behalf.</p>
<p>Typically, visiting the Pages settings page is enough to fix this since we do probe your DNS configuration at that time and issue a new certificate when we validate we can secureyour <code>www</code> variant.</p>
<p>I went ahead and cleared + requested a brand new certificate for you. It should be available within ~ 10 minutes. Let me know if that helps.</p>

**Comment by whalley** [comment]
<p>Great. Works, thanks.</p>
